c++/src/kj/compat/tls-test.c++

```diff
@@ -637,7 +637,10 @@ KJ_TEST("TLS full duplex") {
 
   auto writeUp = writeN(*client, "foo", 10000);
   auto readDown = readN(*client, "bar", 10000);
+#ifndef _WIN32
+  // TODO(soon): work out why this expectation fails even with the above fix
```

Comment:
Apparently it only fails in your particular configuration, since it passes on our Windows CI. Can we narrow the check to whatever's unusual in your case rather than weaken all the tests on Windows?